src/tests/fuzz/README

```diff
@@ -0,0 +1,51 @@
+How to run fuzzing ?
+
+1) With oss-fuzz
+- install docker
+- run git clone --branch suricata --depth 1 https://github.com/catenacyber/oss-fuzz
+(we will use the original google repo once we merge this)
+- change directory into cloned repository : cd oss-fuzz
+- run python infra/helper.py build_image suricata
+- run python infra/helper.py build_fuzzers --sanitizer address suricata
+You can use undefined sanitizer (memory sanitizer does not work yet see https://github.com/google/oss-fuzz/issues/2145#issuecomment-485781098
+- run python infra/helper.py run_fuzzer suricata fuzz_siginit
```

Comment:
I followed these instructions, and it finds something:
```
#1185462	REDUCE cov: 2233 ft: 6547 corp: 1341/434Kb lim: 1722 exec/s: 10490 rss: 110Mb L: 34/1675 MS: 1 EraseBytes-
	NEW_FUNC[1/2]: 0xac2bf0 in DetectFlowvarSetup /src/suricata/src/detect-flowvar.c:118
	NEW_FUNC[2/2]: 0xac4960 in DetectFlowvarDataFree /src/suricata/src/detect-flowvar.c:79

=================================================================
==12==ERROR: LeakSanitizer: detected memory leaks

Direct leak of 32 byte(s) in 2 object(s) allocated from:
    #0 0x53540d in malloc /src/llvm-project/compiler-rt/lib/asan/asan_malloc_linux.cpp:145:3
    #1 0x7f71fb0bdbea in pcre_get_substring (/lib/x86_64-linux-gnu/libpcre.so.3+0x26bea)

SUMMARY: AddressSanitizer: 32 byte(s) leaked in 2 allocation(s).
INFO: to ignore leaks on libFuzzer side use -detect_leaks=0.

MS: 2 InsertByte-CMP- DE: "flowvar"-; base unit: 57e465d2d1845493b38ad94c29c253cafa06e2f7
artifact_prefix='./'; Test unit written to ./leak-891cafa9d757a49bd62f2dc290554ad3143595f7
```
What is the next step?

Reply:
> What is the next step?

Fix the bug :-)
You can use the reproducer file named `leak-891cafa9d757a49bd62f2dc290554ad3143595f7`
Then, it is up to the debugging skills

Reply:
Right, found the input file and opened a ticket https://redmine.openinfosecfoundation.org/issues/3489